packages/mobile/src/home/NotificationBox.tsx

```diff
@@ -61,222 +68,217 @@ export enum NotificationBannerCTATypes {
-            },
+        },
+        {
+          text: t('global:dismiss'),
+          onPress: () => {
+            ValoraAnalytics.track(HomeEvents.notification_select, {
+              notificationType: NotificationBannerTypes.celo_asset_education,
+              selectedAction: NotificationBannerCTATypes.decline,
+            })
+            dispatch(dismissGoldEducation())
           },
-        ],
-      })
-    }
+        },
+      ],
+    })
+  }
 
-    if (!dismissedInviteFriends && !pausedFeatures.INVITE) {
-      actions.push({
-        text: t('inviteFlow11:inviteAnyone'),
-        icon: inviteFriends,
-        callToActions: [
-          {
-            text: t('global:connect'),
-            onPress: () => {
-              dispatch(dismissInviteFriends())
-              ValoraAnalytics.track(HomeEvents.notification_select, {
-                notificationType: NotificationBannerTypes.invite_prompt,
-                selectedAction: NotificationBannerCTATypes.accept,
-              })
-              // TODO: navigate to relevant invite flow
-            },
-          },
-          {
-            text: t('global:remind'),
-            onPress: () => {
-              dispatch(dismissInviteFriends())
-              ValoraAnalytics.track(HomeEvents.notification_select, {
-                notificationType: NotificationBannerTypes.invite_prompt,
-                selectedAction: NotificationBannerCTATypes.decline,
-              })
-            },
-          },
-        ],
-      })
-    }
+  return actions
+}
+
+function useNotifications() {
+  const notifications: Notification[] = []
 
-    return actions.map((notification, i) => <SimpleMessagingCard key={i} {...notification} />)
+  // Pending outgoing invites in escrow
+  const reclaimableEscrowPayments = useSelector(getReclaimableEscrowPayments)
+  if (reclaimableEscrowPayments && reclaimableEscrowPayments.length) {
+    notifications.push({
+      element: (
+        <EscrowedPaymentReminderSummaryNotification key={1} payments={reclaimableEscrowPayments} />
+      ),
+      priority: INVITES_PRIORITY,
+    })
+  }
+
+  // Incoming payment requests
+  const incomingPaymentRequests = useSelector(getIncomingPaymentRequests)
+  if (incomingPaymentRequests && incomingPaymentRequests.length) {
+    notifications.push({
+      element: (
+        <IncomingPaymentRequestSummaryNotification key={1} requests={incomingPaymentRequests} />
+      ),
+      priority: INCOMING_PAYMENT_REQUESTS_PRIORITY,
+    })
+  }
+
+  // Outgoing payment requests
+  const outgoingPaymentRequests = useSelector(getOutgoingPaymentRequests)
+  if (outgoingPaymentRequests && outgoingPaymentRequests.length) {
+    notifications.push({
+      element: (
+        <OutgoingPaymentRequestSummaryNotification key={1} requests={outgoingPaymentRequests} />
+      ),
+      priority: OUTGOING_PAYMENT_REQUESTS_PRIORITY,
+    })
   }
 
+  const notificationsConfig = useNotificationsConfig()
```

Comment:
nit: Would there be a better name? 
```suggestion
  const simpleActions = useSimpleActions()
```

Reply:
Yeah, I changed actions because we don't call these actions, they are notifications, but I agree that notifications config is not great either and I don't have better ideas, so I'll go with your suggestion